src/polymer/polymer2-element-scanner.ts

```diff
@@ -188,7 +188,7 @@ export class Polymer2ElementScanner implements JavaScriptScanner {
         description: (docs.description || '').trim(),
         superClass: this._getExtends(node, docs, warnings, document),
         mixins: jsdoc.getMixins(document, node, docs, warnings),
-        privacy: getOrInferPrivacy(className || '', docs, false),
+        privacy: getOrInferPrivacy(className || '', docs, false, 'private'),
```

Comment:
> unless they're part of a namespace, or they're defined elements
Is this easy to analyse statically, though? 

So, any proposed  changes? Should we leave it at private?